database/models/schedule_models.py

```diff
@@ -65,6 +65,25 @@ def __str__(self):
                + " Section " + self.section_id
 
 
+class SectionNote(models.Model):
+    COLOR_RED = 1
+    COLOR_GREEN = 2
+    COLOR_BLUE = 3
+
+    COLOR_CHOICES = [
```

Comment:
Do you think you could re-do this using https://pypi.org/project/django-colorfield/
I think this functionality will be more useful if the user is not limited to 3 colors

Reply:
Everything looks great otherwise